lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/exploring-mpg.adoc

```diff
@@ -0,0 +1,78 @@
+= Exploring the Fuel Efficiency Dataset
+++++
+<style>
+body.workbookpage td .autonum::after { content: ')' !important; }
+</style>
+++++
+For this page, you'll need to the  @starter-file{alg2-fuel-efficiency} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+@n Click "Run", and evaluate `mpg-table` in the Interactions Area. Do you know what each column means?
+
+[cols="1a,1a", grid="none", frame="none", stripes="none"]
+|===
+|
+- *test* - the number of the fuel efficiency test
+- *model* - the car used in the test (A-F)
+|
+- *speed* - the speed at which the car was tested
+- *mpg* - the miles-per-gallon the car achieved at that speed
+|===
+
+@n Evaluate `A15`, `A45` and `A75` in the Interactions Area. What *model* of car is used in all three rows? @fitb{}{@ifsoln{`A`}}
+
+@n At what *speeds* is this model being tested in each of the rows? @fitb{}{@ifsoln{15, 55, and 75 miles per hour}}
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="1a,1a", options="frame",stripes="none"]
+!===
+| @n In Pyret, make a scatter plot showing the relationship between `speed` and `mpg`, using `model` as your labels, then sketch the resulting scatter plot below.
+|
+--
+Based on the scatter plot...
+
+@n What do you *Notice?* @fitb{}{} +
+@fitb{}{} +
+@fitb{}{}
+
+@n What do you *Wonder?* @fitb{}{} +
+@fitb{}{} +
+@fitb{}{}
+
+@n What speed is generally the most fuel-efficient? @fitb{}{@ifsoln{45-55mpg}}
+
+@n Which model is generally the most fuel-efficient? @fitb{}{@ifsoln{B}}
+
+@n Which model is generally the _least_ fuel-efficient? @fitb{}{@ifsoln{G}}
+--
+!===
+
+@n Do you see a correlation? Describe its form (linear/nonlinear/none), direction (positive/negative), and strength (strong/moderate/weak/none):
```

Comment:
Wonder if this would make more sense with one of those sentences with fitbruby

Either way, this could use some word smithing. it's a little strange to bother asking whether they see a correlation and then proceed with the assumption that they have.

Reply:
I don't think a fitbruby is appropriate here, since it overly scaffolds and boxes kids into choosing correlations with a direction. I did some wordsmithing though -- let me know if it looks better?